### PR TITLE
Remove friendly_id monkey patches

### PR DESCRIPTION
**Pull request by tekin**
The sequential slugging code is now part of the latest beta release of friendly_id, meaning the custom slug generating code can be removed.

### CODE REVIEW

File: config/initializers/friendly_id.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
-  config.use :slugged, :finders, FriendlyId::SequentialSlugs
+  module CustomNormalisation
+    def normalize_friendly_id(input)
+      # Use the babosa gem for better string parameterisation
+      super input.to_s.to_slug.truncate(150).normalize.to_s
+    end
+  end
 
+  config.base = :name
+  config.use :sequentially_slugged, :finders, CustomNormalisation
```
Comment (boffbowsh) [inline config/initializers/friendly_id.rb]: Are we using `:finders`? In the work to upgrade to 4.2.2 I changed all the calls to be `Model.friendly.find`.

Comment (tekin) [inline config/initializers/friendly_id.rb]: If all the calls were changed, then theoretically, the `:finders` module is no longer required. That feels like a separate concern to the changes in this pull request though - the above changes don't add it, they merely maintain it.

Comment (boffbowsh) [inline config/initializers/friendly_id.rb]: Yeah, fair enough.

**Comment by boffbowsh** [comment]
This all makes sense, but would we be better off waiting until `friendly_id` is out of beta?

File: config/initializers/friendly_id.rb (hunk 1 above)
Comment (tekin) [inline config/initializers/friendly_id.rb]: I just pushed a branch with the finders add-on removed and it looks like there are still a handful of calls to the old magic finders: https://ci-new.alphagov.co.uk/job/govuk_whitehall_branches/6237/console

**Comment by tekin** [comment]
Possibly. I wanted to get a head start on this before I finish with GDS to make sure it got done as I'm the one most familiar with the sequential slugging patches. I've reviewed the changes and apart from the sequential slugging code itself, they are fairly minimal: https://github.com/norman/friendly_id/compare/5.1.0...ff72c48

File: config/initializers/friendly_id.rb (hunk 1 above)
Comment (boffbowsh) [inline config/initializers/friendly_id.rb]: Eurgh. But I had to fix loads of them, wth?

**Comment by boffbowsh** [comment]
Ok, well in that case, :+1:

**Comment by tekin** [comment]
Having thought about it, it might be best to wait for the final gem to be released, It doesn't sound like it's far off. Will re-open when it has.

**Pull request closed without merging**